react/features/authentication/middleware.js

```diff
@@ -114,7 +115,16 @@ MiddlewareRegistry.register(store => next => action => {
                 && error.name === JitsiConnectionErrors.PASSWORD_REQUIRED
                 && typeof error.recoverable === 'undefined') {
             error.recoverable = true;
-            store.dispatch(_openLoginDialog());
+
+            // SONRISA MODIFICATION: instead of show login form, login user with credentials set by native app
+            const { jid, password } = _getCredentials(store.getState());
+
+            if (jid && password) {
+                logger.info('Try to login with JID', jid);
```

Comment:
Is this log message useful now that you know the code works? Seems like it's a debug log. My problem with it is that here, only an action dispatch happens, we're not actually trying to log in here.